docs/index.md

```diff
@@ -58,19 +58,19 @@ A directory of documentation and source repositories for the Secure Device Onboa
 
 ## The Secure Device Onboard Process
 
-This section describes a typical process in which Secure Device Onboard is used to automatically and securely onboard a device onto an owners network.  Those process include:
```

Comment:
It appears that the plural demonstrative  "those"  is modifying the singular noun.  So changing process to processes.